### PR TITLE
new API call testing

### PR DESCRIPTION
**Pull request by kimberlyjk**
This PR contains one new test for the new API call and related functions I made in the Pull Request #7 .

Minor Changes to the following files:  
- babel config -- configurational changes
- client/overview -- reorganized when to fetch overview information from the server and made minor tweaks to html and React code
- database/index.js -- minor changes to names of files, variable references and commenting of code. 
- database/seeder.js - changed up the null references to be empty strings
- pack.lock.json and package.json -- addition of new npm packages needed for testing

major change to on file _**testing/api.spec.js**_ -- not sure why the entire file looks brand new, might have to do with where I cut the branch from. 
- added the last test _/overview/all should fetch object with expected properties per app and service plan_
- in order to run test you can download the branch, npm install and run `npm run test`

### CODE REVIEW

File: client/overview.jsx [hunk 1]
```diff
@@ -1,44 +1,56 @@
...
+        owner: overview.owner
       });
-      return overview;
+      // console.log(this.state)
     });
   }
 
   componentDidMount() {
-    this.fetchOverview();
+    // this.fetchOverview();
```
Comment (djian001) [inline client/overview.jsx]: do you still need this if you're doing your request in your constructor?

File: client/Components/amenities.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Amenities extends React.Component {
```
Comment (djian001) [inline client/Components/amenities.jsx]: Are you planning on changing the state for these?

File: database/helperFuncs.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+
+module.exports = {
```
Comment (djian001) [inline database/helperFuncs.js]: this is cool!

File: database/seeder.js [hunk 4]
```diff
@@ -199,7 +199,7 @@ const seed = async () => {
           potableWater: {
             available: false,
             types: 0,
-            description: null
+            description: ''
```
Comment (djian001) [inline database/seeder.js]: How come you decided to change this from null to a string?

**Review by djian001 — APPROVED** [review]
love the helperFuncs.js file! didn't realize you had so many of these functions!

File: client/overview.jsx (hunk 1 above)
Comment (kimberlyjk) [reply client/overview.jsx]: no, I think I updated this in subsequent code, but good catch!

File: client/Components/amenities.jsx (hunk 2 above)
Comment (kimberlyjk) [reply client/Components/amenities.jsx]: probably. I think setting state might be easier than just working with the props... but I am still 100% sure what my intentions are... lol

File: database/seeder.js (hunk 4 above)
Comment (kimberlyjk) [reply database/seeder.js]: This was for that "graceful error handling" that Leslie was talking about. I was noticing that my app would just die when it was seeing null in lieu of an empty string.... but I haven't figured it out fully yet.

**Pull request merged**